src/ray/core_worker/task_manager.cc

```diff
@@ -28,6 +28,26 @@ const int64_t kTaskFailureThrottlingThreshold = 50;
 // Throttle task failure logs to once this interval.
 const int64_t kTaskFailureLoggingFrequencyMillis = 5000;
 
+TaskStatusCounter::TaskStatusCounter() {
+  // TODO(ekl): this isn't actually resetting OpenCensus properly
```

Comment:
Is the issue because driver state is not resetted properly (the task you assigned to me)?

Reply:
Actually, this workaround doesn't work either (I'll remove this code).